openmc/mgxs/groups.py

```diff
@@ -299,3 +303,127 @@ def merge(self, other):
         # Assign merged edges to merged groups
         merged_groups.group_edges = list(merged_edges)
         return merged_groups
+
+
+class DelayedGroups(object):
```

Comment:
So a high level question - why is a `DelayedGroups` class needed? It seems like it is effectively just a list of integers without any unique functionality. What is your reasoning for creating a class rather than simply storing the list of integers directly to the `MDGXS.delayed_groups` attribute?